src/libtriton/includes/triton/pathManager.hpp

```diff
@@ -88,7 +88,7 @@ namespace triton {
           TRITON_EXPORT void pushPathConstraint(const triton::arch::Instruction& inst, const triton::engines::symbolic::SharedSymbolicExpression& expr);
 
           //! Pushes constraint created from node to the current path predicate.
-          TRITON_EXPORT void pushPathConstraint(const triton::ast::SharedAbstractNode& node);
+          TRITON_EXPORT void pushPathConstraint(const triton::ast::SharedAbstractNode& node, const std::string& comment);
```

Comment:
As a matter of homogeneous coding style, can you define the default parameter in `.hpp` and not `.cpp`?